lib/tasks/one_off.rake

```diff
@@ -138,4 +138,35 @@ namespace :pvb do
     FileUtils.rm(password_filename)
     STDOUT.puts "\nRemoved passwords."
   end
+
+  desc 'Backpopulate cancellation reasons'
+  task backpopulate_cancellation_reasons: :environment do
+    missing_cancellation = Visit.
+                           preload(:cancellation).
+                           with_processing_state('cancelled').
+                           joins(<<-EOS).
+        LEFT OUTER JOIN cancellations \
+          ON cancellations.visit_id = visits.id
+      EOS
+                           where(cancellations: { id: nil })
+
+    batch = missing_cancellation.limit(1).to_a
```

Comment:
Oh, right: because you're getting a new `batch` later… This seems confusing; what's the benefit over getting them all here?